src/components/AnonymousReportFooter.tsx

```diff
@@ -30,9 +33,9 @@ function AnonymousReportFooter({isSmallSizeLayout = false, personalDetails, repo
             <View style={[styles.flexRow, styles.flexShrink1]}>
                 <AvatarWithDisplayName
                     report={report}
-                    personalDetails={personalDetails}
                     isAnonymous
                     shouldEnableDetailPageNavigation
+                    policy={policy}
```

Comment:
I can explain. Previously, we were passing the `policies` prop to `ReportFooter` and it has `ReportAnonymousFooter` as a child so it re-renders whenever `policies` update. But since I removed that altogether, it was not updating in real-time anymore. So I decided to use the dedicated `policy` in the nearest component possible.